py/desispec/scripts/stdstars.py

```diff
@@ -81,6 +81,23 @@ def main(args) :
     log.info("mag delta %s = %f (for the pre-selection of stellar models)"%(args.color,args.delta_color))
     log.info('multiprocess parallelizing with {} processes'.format(args.ncpu))
 
+    # READ DATA
+    ############################################
+    # First loop through and group by exposure and spectrograph
+    frames_by_expid = {}
+    for filename in args.frames :
+        log.info("reading %s"%filename)
+        frame=io.read_frame(filename)
+        header=fits.getheader(filename, 0)
```

Comment:
`frame.meta` has the header from HDU 0 of the frame file read in at the same time as everything else.  Let's use that to avoid another file access.

Reply:
Moving offline discussion here for the future:  this header read in is also done for sky and flat frames, which have similar sky.header and flat.header  attributes. So I'm removing all three of these calls to fits.getheader() and using those attributes that are already in memory.